src/xcode/ENA/ENA/Source/Scenes/TicketValidation/TicketValidationCoordinator.swift

```diff
@@ -131,11 +134,11 @@ final class TicketValidationCoordinator {
 	) {
 		let secondConsentViewController = SecondTicketValidationConsentViewController(
 			viewModel: SecondTicketValidationConsentViewModel(
-				// EXPOSUREAPP-10834: Needs to be set from allowlist when it's available: allowlist[0].serviceProvider
 				serviceIdentity: "",
```

Comment:
My expectation was that we just need to do what's written in the comment: pass in `ticketValidation.allowList.validationServiceAllowList.first?.serviceProvider ?? ""`. You need to make the `allowList` part of the protocol, but that's it. The allowList is already fetched and available here, isn't it?

Reply:
no it is fetched inside the TicketValidation not the coordinator